receiver/receivercreator/README.md

```diff
@@ -68,33 +96,37 @@ targeting it will have different variables available.
 |-------------|-----------------------------------|
 | type.pod    | `true`                            |
 | name        | name of the pod                   |
+| namespace   | namespace of the pod              |
+| uid         | unique id of the pod              |
 | labels      | map of labels set on the pod      |
 | annotations | map of annotations set on the pod |
 
 ### Port
 
-| Variable        | Description                          |
-|-----------------|--------------------------------------|
-| type.port       | `true`                               |
-| name            | container port name                  |
-| port            | port number                          |
-| pod.name        | name of the owning pod               |
-| pod.labels      | map of labels of the owning pod      |
-| pod.annotations | map of annotations of the owning pod |
-| protocol        | `TCP` or `UDP`                       |
+| Variable        | Description                             |
+|-----------------|-----------------------------------------|
+| type.port       | `true`                                  |
+| name            | container port name                     |
+| port            | port number                             |
+| protocol        | The transport protocol ("TCP" or "UDP") |
+| pod.name        | name of the owning pod                  |
+| pod.namespace   | namespace of the pod                    |
+| pod.uid         | unique id of the pod                    |
+| pod.labels      | map of labels of the owning pod         |
+| pod.annotations | map of annotations of the owning pod    |
 
 ### Host Port
 
-| Variable       | Description                                      |
-|----------------|--------------------------------------------------|
-| type.hostport  | `true`                                           |
-| name           | Name of the process                              |
-| command        | Command line with the used to invoke the process |
-| is_ipv6        | true if endpoint is IPv6, otherwise false        |
-| port           | Port number                                      |
-| transport      | The transport protocol ("TCP" or "UDP")          |
+| Variable      | Description                                      |
+|---------------|--------------------------------------------------|
+| type.hostport | `true`                                           |
+| name          | Name of the process                              |
```

Comment:
Should `name` be called `process`, `process_name` or `process.executable.name` (the Otel convention) instead?

Reply:
`process_name` seems better (we don't use otel conventions since the prefixes would be mostly redundant).